### PR TITLE
Desktop version Design

### PR DESCRIPTION
**Pull request by Kaghenimbale**
- [ ]  Use media queries in your CSS file to make your desktop version(Desktop: 768px or wider).
- [ ]  Use Grid
- [ ]  Use Flexbox

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -377,3 +374,237 @@ fieldset {
...
+
+  .nav-link li a {
+    font-size: 1.3rem;
+    padding-left: 5rem;
+    color: #344563;
+  }
+
+  .section-1 {
+    padding: 0 10rem 0 5rem;
+    width: 80vw;
```
Comment (muneebulrehman) [inline style.css]: Great job with the project so far. However, setting a fixed width for this section is causing unusual behavior on larger screens. I have recorded [this video](https://www.loom.com/share/f5e07483e641494b84a8d37b17b2fa51) where I tried to explain exactly what the issue is. Kindly fix the issue and then submit for the re-review.

File: style.css [hunk 2]
```diff
@@ -377,3 +374,237 @@ fieldset {
...
+    width: 50%;
+    gap: 1rem;
+  }
+
+  .about-2 {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+  }
```
Comment (muneebulrehman) [inline style.css]: As per the Figma template of your choosing, these two sections are not of equal width. The first `about section` is about 40% and the other about section is about 60%. I have recorded [this video](https://www.loom.com/share/f5e07483e641494b84a8d37b17b2fa51) where I compared your UI with that of the Figma template and explained the fix for it. Kindly fix this issue and then submit it for a re-review.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Kaghenimbale , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Good docs.

## Required Changes ♻️
_Check the comments under the review._

- [ ] The button should have four interaction stages. ⬇️
image
However, you only have implemented `hover` and `enabled`. Kindly implement other two interactions also. You can watch [this video](https://www.loom.com/share/f5e07483e641494b84a8d37b17b2fa51) where I demonstrated the problem.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Kaghenimbale , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**